doc/source/running.rst

```diff
@@ -184,8 +184,25 @@ restore.
 
 .. include:: .res/code-block/bash/pbm-restore-mongodb-uri.txt
 
-After a cluster's restore is complete all mongos nodes will need to be
-restarted to reload the sharding metadata.
+After a cluster's restore is complete, restart all ``mongos`` nodes to reload the sharding metadata.
+
+.. rubric:: Restoring a large backup
+
+When you restore a backup that is large for the currently available memory on the server, the restore operation may fail with the out of memory error. To avoid that error, you can adjust the memory consumption during restore. Define the following options in the |PBM| config:
```

Comment:
Current phrasing sets user expectations to the error which could happen only on the setups with tight memory restrictions. I'd rather put that in a way that "these new options are introduced for better adjustments in environments with tight memory bounds". Also maybe worth to mention that defaults were changed to fit setups with 1Gb and less for the agent.

Reply:
@akira-kurogane what do think? ^^^

Reply:
Seems to be the same comment repeated three times. To cut-and-paste what I replied to the first comment:

_Yes, it's not how big the backup is - the database could be 10GB or 10TB. The issue that happened was caused by the amount of memory that will be in use by pbm-agents_ at any given instant _vs. the RAM available. It seems that could be as high as 2GB, but you can tune the rate of IO to be lower with this options in this patch, and that really reduces the instantaneous memory needs. And thus avoid OOM death._

_The wording "these new options are introduced for better adjustments in environments with tight memory bounds" is good i.m.o._

Reply:
yep, it duplicated